benchmark/torch/qmix/README.md

```diff
@@ -0,0 +1,31 @@
+## QMIX based on Pytorch
+We reproduce the QMIX based on **PARL** and **Pytorch**, reaching the same level of indicators as the paper in StarCraftII benchmarks.
+### QMIX
+QMIX is a **value-based** multi-agent reinforcement learning algorithm.  
+See more information about QMIX in: [QMIX: Monotonic Value Function Factorisation for Deep Multi-Agent Reinforcement Learning](https://arxiv.org/abs/1803.11485)
+### StarCraftII Environment
+Paper: [The StarCraft Multi-Agent Challenge](https://arxiv.org/pdf/1902.04043)  
+Github Repositories: [smac](https://github.com/oxwhirl/smac)  
+## Benchmark Results
+- We traineded our model in 5 different scenarios: *"3m", "8m", "2s_3z", "3s_5z"* and *"1c_3s_5z"*.  
+- The **difficulty** in all scenarios are set to be "7" (very difficult).  
+
+<img src="images/qmix_result.png" width = "1500"  alt="Performance" />
```

Comment:
We should run at least three experiments for each environment.

Reply:
done.